news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
+date: 2017-11-05
+--- 
+
```

Comment:
you need a short summary of your article as the opener, the things before the banner are loaded like ![this](https://puu.sh/yaPhB/252c130ab6.png)

on the new page so this sentence being missing will just break al ot of stuff

Reply:
Gonna add it, will have to figure out where though.